### DISCUSSION TITLE
get current pageNumber and totalPages in Blade View

### DISCUSSION THREAD

**Discussion by zaidpirwani** [spatie/laravel-pdf#208] [Q&A]
Hello,

Is there any way to get Current Page Number and Total Pages inside the main Blade View - I can get them in footer view

I want to modify the first page depending on if there is one page or multiple

if I use @pageNumber or @totalPages in the main blade View, I dont get any output.

**Comment by fesf-it-manager** [comment]
I realize that the page number is only available in header and footer views, and possibly this is dependent on how browsershot uses puppeteer which runs chrome to generate the pdf and the page number is generated and available there, not available in the main blade.

It will be very convenient if somehow we can get this data in blade - I need to generate different header/footer based on page numbers and I also need to modify page content based on page number - but maybe not possible in the current setup / system
